src/relayclient/RelayClient.ts

```diff
@@ -108,16 +139,18 @@ export class RelayClient {
   }
 
   async _init (): Promise<void> {
+    if (this.initialized) { return }
+    this.emit(new GsnInitEvent())
```

Comment:
at least do emit after the asynchronous operation

Reply:
I do the events before the action (with the exception of "refresh", where the "done" event carries some info) . the idea that the event title means we're in this state.
anyway, I think this event will go away, since the "init" should be ran always after construction, not before first relayTransaction